pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
Nope, you're still not seeing my concern.  You're still telling me things I know.

I am deeply intimate with the internal workings of the PIC32 interrupt system.

Can you actually see the case that I am concerned with? Can you describe it to me?